packages/affiliates/liquidity-mining/ugas-jan21-weekly-payouts/Week_2_Mining_Rewards.json

```diff
@@ -0,0 +1,25 @@
+{
+  "week": 2,
+  "fromBlock": 11412831,
+  "toBlock": 11458320,
```

Comment:
to and from block are different than what we ran for the other scripts. We ran these from 2020-12-07 23:00:00 UTC to 2020-12-14 23:00:00 UTC. Startblock 11408486 and end block 11453986.

Just as FYI you can see the block numbers in the deployer rewards output file under "startBlock" and "endBlock"

Reply:
Thanks for pointing this out David. I reran the calculation with startblock 11408486 and end block 11453986 to match the developer mining window.

This happened because the previous uGAS LM calculation was run with a slightly incorrect end-bound on the timeframe last week. Approximately 16 hours of time was factored into the LM calculation that was not part of the uGAS developer mining rewards. This was factored into the LM distribution calculation for last week, but obviously did not affect the total amount of rewards that was distributed. Effectively those 16 hours of developer mining rewards have not been paid out yet, so it seems best to align the LM calculation with the developer mining calculation this week as you suggested.